js/dialogcards.js

```diff
@@ -476,11 +661,50 @@ H5P.Dialogcards = (function ($, Audio, JoubelUI) {
       self.stopAudio(self.$current.index());
       self.$current.removeClass('h5p-dialogcards-current');
       self.$current = $prev.addClass('h5p-dialogcards-current').removeClass('h5p-dialogcards-previous');
+      self.turnCardToFront ();
       self.updateNavigation();
     }
   };
 
   /**
+   * User selected cards order option (normal/random).
+   */
+  C.prototype.randomizeOrder = function (randomizeYesNo) {
+    var self = this;
+    self.randomCards = randomizeYesNo;
+    var $el = '.h5p-dialogcards-order';
```

Comment:
A string shouldn't be prefixed with a $. $ prefix in jQuery code usually means that the variable contains a jQuery object.